pages/casestudies/[id].js

```diff
@@ -81,6 +81,16 @@ function Index({
           <Heading typeStyle="heading-xl" className="countdown-text-gradient">
             {casestudy.company.name}
           </Heading>
+          <div >
+            <Heading typeStyle="body-lg">
+                    <Link href={casestudy.company.contact[0].link}>
+                      <a className="  text-md leading-5 font-medium text-gray-900 
+                      hover:underline" target="_blank">
+                        {casestudy.company.contact[0].name}
```

Comment:
@derberg if this parameter has been used as `casestudy.company.contact[0].name`, can there be a possibility that there are 2 or more contacts inside a case study? If yes, then this implementation shouldn't be hardcoded this way.

Reply:
Yes, I feel the same.
We should do something with this one.